docs/source/development/defining_new_models.md

```diff
@@ -0,0 +1,216 @@
+# How to add a new model to the Virtual Rainforest
+
+The Virtual Rainforest is designed to be modular, meaning that the set of models to be
+used in a particular run is configurable at the start of the simulation. We are starting
+out by defining a core set of models (`abiotic`, `animals`, `plants` and `soil`), which
+will generally all be included for the vast majority of simulations. In future, it might
+be desirable to include models for other aspects of rainforests (e.g. `freshwater`), or
```

Comment:
```suggestion
be desirable to include models for other aspects of rainforests (e.g. `freshwater`, `disturbance`), or
```

Reply:
I think it might be worth mentioning disturbance as this will be one of the major applications